edelphi/src/main/java/fi/metatavu/edelphi/query/thesis/TimelineThesisQueryPageHandler.java

```diff
@@ -240,6 +244,11 @@ public void exportData(QueryExportContext exportContext) {
       }
     }
   }
+
+  @Override
+  public ReportPageCommentProcessor exportComments(QueryPage queryPage, PanelStamp stamp, List<QueryReply> replies) {
+    return new TimelineReportPageCommentProcessor(stamp, queryPage, new HashMap<>());
+  }
   
   @Override
   protected void renderReport(PageRequestContext requestContext, QueryPage queryPage) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this method to reduce its Cognitive Complexity from 38 to the 15 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3776)